### PR TITLE
Added phone skin to designer

### PR DESCRIPTION
**Pull request by Amerkashi**
Added phone skin to designer.
image

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java [hunk 1]
```diff
@@ -292,17 +297,37 @@ public MockForm(SimpleEditor editor) {
...
     // It will definitely be modified in the future to add more options.
     PORTRAIT_WIDTH = width;
     PORTRAIT_HEIGHT = height;
     LANDSCAPE_WIDTH = height;
     LANDSCAPE_HEIGHT = width;
 
-    if (landscape)
+//    if (isTablet) {
+//      PORTRAIT_WIDTH = TABLET_PORTRAIT_WIDTH;
+//      PORTRAIT_HEIGHT = TABLET_PORTRAIT_HEIGHT;
```
Comment (ewpatton) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Remove commented out code

File: appinventor/appengine/war/Ya.css [hunk 2]
```diff
@@ -990,6 +990,65 @@ select.ode-PropertyEditor[disabled] {
   border: 2px solid black;
 }
 
+
+.ode-SimpleMockFormPhoneLandscape {
+    background: url(images/phoneLandscape.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 100% 100%;
+    /*padding: 5px 54px 5px 54px;*/
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: Remove commented out styles here and elsewhere in this patch.

File: appinventor/appengine/war/Ya.css [hunk 3]
```diff
@@ -990,6 +990,65 @@ select.ode-PropertyEditor[disabled] {
...
+.ode-SimpleMockFormPhonePortraitMonitor {
+  background: url(images/phonePortraitMonitor.png);
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: 100% 100%;
+  /*padding: 5px 55px 5px 54px;;*/
+  /*padding: 11px 54px 11px 64px;*/
+  padding: 99px 36px 75px 34px;
+}
+/**/
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: What are the purposes of these empty comments?

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java (hunk 1 above)
Comment (Amerkashi) [reply appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Done

File: appinventor/appengine/war/Ya.css (hunk 2 above)
Comment (Amerkashi) [reply appinventor/appengine/war/Ya.css]: Done

File: appinventor/appengine/war/Ya.css (hunk 3 above)
Comment (Amerkashi) [reply appinventor/appengine/war/Ya.css]: Added start / end comment-tag to show code-updates for phone skins

**Comment by kkashi01** [comment]
@ewpatton @jisqyv @SusanRatiLane Any other changes needed?

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java [hunk 4]
```diff
@@ -521,10 +535,12 @@ private void setScreenOrientationProperty(String text) {
         screenWidth = LANDSCAPE_WIDTH;
         screenHeight = LANDSCAPE_HEIGHT;
         landscape = true;
+        phoneWidget.setStyleName("ode-SimpleMockFormPhoneLandscape");
```
Comment (ewpatton) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Need logic similar to changePreviewSize. If you select Monitor sizing, and then switch to landscape, the image that shows is the phone image scaled way up.

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java [hunk 5]
```diff
@@ -521,10 +535,12 @@ private void setScreenOrientationProperty(String text) {
...
       } else {
         screenWidth = PORTRAIT_WIDTH;
         screenHeight = PORTRAIT_HEIGHT;
         landscape = false;
+        phoneWidget.setStyleName("ode-SimpleMockFormPhonePortrait");
```
Comment (ewpatton) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Ditto above comment.

Comment (Amerkashi) [reply appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Code update pushed

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java (hunk 4 above)
Comment (Amerkashi) [reply appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: Code update pushed

**Pull request merged**